go/tao/auth/lexer.go

```diff
@@ -81,6 +79,16 @@ var (
 	tokenEOF       = token{itemEOF, nil}
 )
 
+var prinTokens = map[token]bool{
```

Comment:
I think you also need export a way to add prin tokens, or you need to accept any lower-case value in the right context. I prefer the former option.

Could you then please add a test file that has a different package ("auth_test" rather than "auth") and make sure you can add new prin tokens and use them in a policy?